### ISSUE TITLE
Stack.horizontal = False does not work

### ISSUE THREAD

**Issue by mikaelho** [pglet/pglet-python#46]
I would expect `horizontal=False` to have the same effect as leaving the parameter out. This would enable having a boolean switch to control this.

In practice `horizontal=False`, looks the same as `horizontal=True`, see this [repl](https://replit.com/@mikaelho/pglet-stack-horizontal).

This seems to be a bug on the server side, as the Python code looks the same for this boolean attribute as for every other one - or then all the boolean attributes would have this issue, which seems unlikely.

**Comment by FeodorFitsner** [comment]
Indeed, there is a bug. Thanks for reporting it. Will release a patch shortly.
